com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1818,7 +1818,7 @@ private List<TaskResult> runTasks(IProgress monitor) throws IOException {
                 }
 
                 TimeProfiler.start(task.getName());
-                TimeProfiler.addData("output", task.getOutputsString());
+                TimeProfiler.addData("output", StringUtil.truncate(task.getOutputsString(), 1000));
```

Comment:
this one isn't directly related to that particular issue, but it's nice to truncate strings which can be huge